web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,38 @@
 <script lang="ts">
-	import { imageLoad } from '$lib/utils/image-load';
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
+	import * as Buffer from 'buffer';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: Buffer;
```

Comment:
Thumbhashes of assets are nullable so at least the `null` type should be included. This component is also used for non-asset thumbnails which don't have a thumbhash, so this prop should also be optional.